<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>541</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Agregar</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>511</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>Neurona</string>
      </property>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>108</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Posicion X</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>108</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Posicion Y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>108</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>108</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>108</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>108</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Voltaje</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>108</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="Id">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="Voltaj">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>62</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="PosicionX">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="PosicionY">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>160</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="Green">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>200</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="Red">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>240</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="Blue">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>280</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>350</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Capturar</string>
       </property>
      </widget>
      <widget class="QLabel" name="Res">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>40</y>
         <width>191</width>
         <height>261</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mostrar</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar Tabla</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>31</y>
        <width>256</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="actionAbrir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
